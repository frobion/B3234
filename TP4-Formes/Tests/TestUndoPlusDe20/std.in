R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
DELETE Segment1
PC PolygonConvex1 1 1 3 -5 -2 -1
R Rectangle2 1 3 6 0
S Segment2 0 0 1 1
DELETE Segment2
PC PolygonConvex2 1 1 3 -5 -2 -1
R Rectangle3 1 3 6 0
S Segment3 0 0 1 1
DELETE Segment3
PC PolygonConvex3 1 1 3 -5 -2 -1
R Rectangle4 1 3 6 0
S Segment4 0 0 1 1
DELETE Segment4
PC PolygonConvex4 1 1 3 -5 -2 -1
R Rectangle5 1 3 6 0
S Segment5 0 0 1 1
DELETE Segment5
PC PolygonConvex5 1 1 3 -5 -2 -1
R Rectangle6 1 3 6 0
S Segment6 0 0 1 1
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
UNDO
LIST
EXIT

