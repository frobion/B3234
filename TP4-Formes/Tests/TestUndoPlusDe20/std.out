OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
ERR
#

